composer.json

```diff
@@ -319,6 +320,10 @@
             "drupal/entity_browser": {
                 "Fix duplicated image selection when click use selected button": "https://www.drupal.org/files/issues/prevent-duplication.patch"
             },
+            "drupal/entity_usage": {
+                "2954973 - Track absolute URLs in link fields": "https://www.drupal.org/files/issues/2021-08-17/2954973-63.patch",
+                "3007752 - Entity usage list does not scale": "patches/entity_usage-usage_list_scale-3007752-17.patch"
```

Comment:
That patch is contributed but in an MR. I just learned that the best practice for those is to copy the patch locally because someone else could change the MR in a way that could break your site.

Reply:
that makes sense. Please add the URL to the comment which holds the patch. Or the URL to the MR.